ui/components/src/StatChart/StatChart.tsx

```diff
@@ -75,6 +77,8 @@ export function StatChart(props: StatChartProps) {
   const availableHeight = height - seriesNameHeight;
   const valueFontSize = useOptimalFontSize({
     text: formattedValue,
+    // override the font size if user selects it in the settings
+    fontSizeOverride: fontSize,
```

Comment:
I'm thinking it might be simpler if we just return font size 

```
    const valueFontSize = fontSize !== undefined ? fontSize : useOptimalFontSize()
```

Reply:
Wait never mind, you cannot conditionally call a hook. Ignore this 😅